nodestream/cli/commands/run_migrations.py

```diff
@@ -0,0 +1,22 @@
+from ..operations import ExecuteMigrations
+from .nodestream_command import NodestreamCommand
+from .shared_options import PROJECT_FILE_OPTION, TARGETS_OPTION
+
+
+class RunMigrations(NodestreamCommand):
+    name = "migrations run"
+    description = "Execute pending migrations on the specified target."
+    options = [PROJECT_FILE_OPTION, TARGETS_OPTION]
+
+    async def handle_async(self):
+        project = self.get_project()
+        migrations = self.get_migrations()
+        targets = self.option("target")
```

Comment:
Can this use the same constant: `TARGETS_OPTION`?

Reply:
Hmm yes we can... fixed.